front/src/Components/EnableCamera/EnableCameraScene.svelte

```diff
@@ -86,7 +95,7 @@
             <img class="background-img" src={cinemaCloseImg} alt="" />
         </div>
     {/if}
-    <HorizontalSoundMeterWidget {stream} />
+    <HorizontalSoundMeterWidget {volume} />
```

Comment:
You can do even better here (I think).

In a Svelte template, you can use the value of store directly by prefixing it with a "$".

So instead of subscribing / unsubscribing to the store, you can write:

```suggestion
    <HorizontalSoundMeterWidget volume={$localVolumeStore} />
```